pyvista/plotting/renderer.py

```diff
@@ -1249,6 +1255,9 @@ def show_bounds(
         # set bounds
         if bounds is None:
             bounds = np.array(mesh.GetBounds())
+        else:
+            bounds = np.array(bounds, dtype=np.float_)
```

Comment:
If `bounds` is already a float64-dtype array (or array subclass) then this will make this line a no-op:
```suggestion
            bounds = np.asanyarray(bounds, dtype=np.float_)
```
I'd also be tempted to just say `dtype=float` because that's shorter, but I guess being explicit is fine too (the end result is the same).